e2e/self_node_remediation_test.go

```diff
@@ -84,7 +84,11 @@ var _ = Describe("Self Node Remediation E2E", func() {
 			pod := findSnrPod(healthyNode)
 			logs, err := utils.GetLogs(k8sClientSet, pod)
 			Expect(err).ToNot(HaveOccurred())
-			logger.Info("logs of healthy self-node-remediation pod", "logs", logs)
+			logger.Info("BEGIN logs of healthy self-node-remediation pod", "pod", pod.GetName())
```

Comment:
looks like the complete AfterEach block can be extracred into a function, it's used twice

Reply:
oh, thank you for notice that!
Yes, it is used completely equal twice _now_, because it should have been  
"logs of healthy self-node-remediation pod" and "logs of **un**healthy self-node-remediation pod" :sweat_smile: 
However, I can get around that and make a function

Reply:
nevermind it has always been "healthy" :see_no_evil: